frontend/public/components/_resource.scss

```diff
@@ -107,6 +107,9 @@ $height: 18px;
   @include co-break-word;
   &__resource-name {
     min-width: 0; // required so co-break-word works
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
This could be a problem in tables. Lots of resources have the same prefix and you wouldn't be able to tell them apart on mobile.

Reply:
@rebeccaalpert Is there a way we can use the ellipsis only in the events view? We can't make this change to resource list tables.